frontend/src/components/note/NoteCard.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+
+import { Card, Divider } from '@/lib/primevue';
+import { userService } from '@/services';
+import { formatDate } from '@/utils/formatters';
+
+import type { Ref } from 'vue';
+import type { Note } from '@/types';
+
+// Props
+type Props = {
+  note: Note;
+  submissionId: string;
+};
+
+const props = withDefaults(defineProps<Props>(), {});
+
+// State
+const cardData: Ref<Note> = ref(props.note);
+const userName: Ref<string> = ref('');
+
+onMounted(async () => {
+  if (props.note.createdBy) {
+    const res = (await userService.searchUsers({ identityId: [props.note.createdBy] })).data;
+    userName.value = res.length ? res.pop().fullName : '';
+  }
+});
+</script>
+
+<template>
+  <Card>
+    <template #title>
+      <h3 class="mt-3 mb-1">{{ cardData.title }}</h3>
+      <Divider type="solid" />
+    </template>
+    <template #content>
+      <div class="grid nested-grid">
+        <!-- Left column -->
+        <div class="col-12 md:col-6 lg:col-4">
+          <div class="grid">
+            <p class="col-12">
+              <span class="key font-bold">Date:</span>
+              {{ cardData.createdAt ? formatDate(cardData.createdAt ?? '') : undefined }}
+            </p>
+          </div>
+        </div>
+        <!-- Middle column -->
+        <div class="col-12 md:col-6 lg:col-4">
+          <div class="grid">
+            <p class="col-12">
+              <span class="key font-bold">Author:</span>
+              {{ userName }}
+            </p>
+          </div>
+        </div>
+        <!-- Right column -->
+        <div class="col-12 md:col-6 lg:col-4">
+          <div class="grid">
+            <p class="col-12">
+              <span class="key font-bold">Category:</span>
+              {{ cardData.noteType }}
+            </p>
+          </div>
+        </div>
+        <div class="col-12 mb-2">
+          {{ cardData.note }}
+        </div>
+      </div>
+    </template>
+  </Card>
+</template>
+
+<style lang="scss">
```

Comment:
Need to use `style scoped` to prevent bleed over to other components. PrimeVue directives will need to use :deep() to properly access them once you do this.

Reply:
Added `scoped` to both NoteCard and PermitCard component.